packages/website/src/components/ErrorsViewer.tsx

```diff
@@ -95,7 +83,7 @@ function ErrorBlock({
         <div>
           {item.suggestions.map((fixer, index) => (
             <div
-              key={index}
+              key={String(index)}
```

Comment:
[Nitpick] Keys can be `number`s, any reason to make this change? (here and the other one)

```suggestion
              key={index}
```

Reply:
hmm, did something changed in react 18?

old doc recommended to use string indexes https://legacy.reactjs.org/docs/lists-and-keys.html#keys